internal/expr/parser.go

```diff
@@ -89,14 +97,19 @@ func (pe *ParsedExpr) String() string {
 	return out.String()
 }
 
+// bypass adds the string between the previous I/O part and the current part.
+func (p *Parser) addBypass() {
```

Comment:
The function name is out-dated in the comment just here :)

Reply:
I've actually removed this function and left `add` as it is, but I have added `addRemainder`.